### PR TITLE
Upgrades: show premium features in business plan "my plan" page

### LINKED ISSUE #5238: Plans: the thank-you and my-plan page for business should include premium features
The business plan thank-you/my-plan page looks pretty anemic right now. It's not showing all of the functionality that we offer as part of the plan.. it's only showing the features unique to the business plan. We should add the premium items too.

### PR DESCRIPTION
**Pull request by artpi**
Fixes #5238 

Shows premium plan features alongside business plan in "my plan" page

zrzut ekranu 2016-05-06 o 16 17 36

CC @rralian @gwwar @adambbecker 

Also:
@mtias who probably will change my copy

### CODE REVIEW

**Comment by mtias** [comment]
I suggest removing the copy altogether :)

File: client/my-sites/plans/current-plan/style.scss [hunk 1]
```diff
@@ -65,3 +65,9 @@
 	font-size: 18px;
 	font-weight: 300;
 }
+
+.current-plan__more-features-header {
+	color: #4f748e;
```
Comment (mtias) [inline client/my-sites/plans/current-plan/style.scss]: Should be a color variable if we are keeping this.

Comment (artpi) [inline client/my-sites/plans/current-plan/style.scss]: Done!

**Comment by artpi** [comment]
> I suggest removing the copy altogether :)

I think its beneficial. If you are upgrading from premium to business, you may be wondering what extra features you are paying for.

plus, fe phrase this in a way, that "now you can do this", but if you have just gone from premium to business, it is not true, you already had this features.
So we need to emphasize which are new and which not.

**Comment by mtias** [comment]
Why should we optimize the display for those that upgraded from one to the other? (I think that should be address anyways in the upgrade flow.) The current split makes me wonder if I have Premium, or if I need to do anything, it adds unnecessary complexity in my opinion.

**Comment by artpi** [comment]
@mtias, I got rid of my awesome copy once again. Happy now? :)
Can we merge this?
zrzut ekranu 2016-05-11 o 09 29 06

**Comment by mtias** [comment]
💥

**Pull request merged**